src/main/resources/spring/migration-bean.xml

```diff
@@ -156,6 +159,10 @@
         <constructor-arg type="java.lang.String" value="index" />
     </bean>
 
+    <bean id="namespaceFile" class="java.io.File">
+        <constructor-arg type="java.lang.String" value="src/test/resources/namespaces.properties"/>
```

Comment:
We probably do not want `test` resources showing up in the production deployment.